Frontend/src/routes.js

```diff
@@ -1,13 +1,12 @@
 import React from "react";
 import { Routes, Route } from "react-router-dom";
-import HomePage from "./pages/HomePage";
-import NotFoundPage from "./pages/NotfoundPage";
+import HomePage from "./pages/ConsumerHistoryPage";
 
 const AppRoutes = () => {
   return (
     <Routes>
       <Route path="/Consumer History" element={<HomePage />} />
-      <Route path="*" element={<NotFoundPage />} />
```

Comment:
Can you leave the notfoundpage , just so it redirects a user to that page and they know theyve entered an invalid page